Error from web-based viewers if using Jinja2<=2.8

The web-based viewer raises a `TemplateAssertionError` if the installed version of `jinja2` <= 2.8. Could a minimum version >=2.9 be added to `setup.py`? Example traceback:

```
Traceback (most recent call last):
  File "/Users/dave/anaconda3/envs/nbdime/lib/python2.7/site-packages/tornado/web.py", line 1510, in _execute
    result = method(*self.path_args, **self.path_kwargs)
  File "/Users/dave/anaconda3/envs/nbdime/lib/python2.7/site-packages/nbdime/webapp/nbdimeserver.py", line 154, in get
    mathjax_config=self.mathjax_config,
  File "/Users/dave/anaconda3/envs/nbdime/lib/python2.7/site-packages/notebook/base/handlers.py", line 424, in render_template
    return template.render(**ns)
  File "/Users/dave/anaconda3/envs/nbdime/lib/python2.7/site-packages/jinja2/environment.py", line 989, in render
    return self.environment.handle_exception(exc_info, True)
  File "/Users/dave/anaconda3/envs/nbdime/lib/python2.7/site-packages/jinja2/environment.py", line 754, in handle_exception
    reraise(exc_type, exc_value, tb)
  File "/Users/dave/anaconda3/envs/nbdime/lib/python2.7/site-packages/nbdime/webapp/templates/nbdimepage.html", line 28, in <module>
    <script id='nbdime-config-data' type="application/json">{{ config_data|tojson|safe }}</script>
TemplateAssertionError: no filter named 'tojson'
```

--- a/setup.py
+++ b/setup.py
@@ -108,7 +108,7 @@ install_requires = setuptools_args['install_requires'] = [
     'requests',
     'GitPython!=2.1.4, !=2.1.5, !=2.1.6',  # For difftool taking git refs
     'notebook',
-    'jinja2',
+    'jinja2>=2.9',
 ]
 
 extras_require = setuptools_args['extras_require'] = {

